<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QDialog" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>sshout.png</normaloff>sshout.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="remote_host_label">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="remote_host_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="remote_port_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">22</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="remote_port_label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="identify_label">
       <property name="text">
        <string>Identify</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="identify_file_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse_identify_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_auto_connect">
     <property name="text">
      <string>&amp;Auto connect</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="settings_pushButton">
       <property name="text">
        <string>&amp;Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_identify_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>browse_identity_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>change_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>start_main_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remote_host_comboBox</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>remote_host_name_change_event(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remote_host_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ConnectionWindow</receiver>
   <slot>remote_host_name_change_event(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_identity_file()</slot>
  <slot>change_settings()</slot>
  <slot>start_main_window()</slot>
  <slot>remote_host_name_change_event(QString)</slot>
  <slot>remote_host_name_change_event(int)</slot>
 </slots>
</ui>
